### PR TITLE
Move to nwb neuroscope

### PR DESCRIPTION
**Pull request by CodyCBakerPhD**
Moved all functions from to_nwb.neuroscope.py and to_nwb.utils.py to this repository as per discussion; still need to thoroughly test setup.py in new environments (will launch another branch once recent changes are through).

I'll also re-analyze some of the function referencing throughout this to improve computational efficiency; if running on lots of data (as we will be soon on the server), even minor time improvements could get magnified.

### CODE REVIEW

File: buzsaki_lab_to_nwb/new-code/yutabehaviordatainterface.py [hunk 1]
```diff
@@ -13,8 +13,7 @@
 from typing import Union
 # TODO: there doesn't seem to be a pypi for to_nwb...
 # we can always have them on our own end locally, but what about users?
-from to_nwb.neuroscope import get_events
-from to_nwb.utils import find_discontinuities, check_module
+from neuroscope import get_events, find_discontinuities, check_module
```
Comment (bendichter) [inline buzsaki_lab_to_nwb/new-code/yutabehaviordatainterface.py:16]:
```suggestion
from .neuroscope import get_events, find_discontinuities, check_module
```

File: buzsaki_lab_to_nwb/new-code/yutalfpdatainterface.py [hunk 2]
```diff
@@ -11,8 +11,7 @@
 # TODO: there doesn't seem to be a pypi for to_nwb...
 # we can always have them on our own end locally, but what about users?
 from ephys_analysis.band_analysis import filter_lfp, hilbert_lfp
-from to_nwb.neuroscope import read_lfp, write_lfp, write_spike_waveforms
-from to_nwb.utils import check_module
+from neuroscope import read_lfp, write_lfp, write_spike_waveforms, check_module
```
Comment (bendichter) [inline buzsaki_lab_to_nwb/new-code/yutalfpdatainterface.py:14]:
```suggestion
from .neuroscope import read_lfp, write_lfp, write_spike_waveforms, check_module
```

File: buzsaki_lab_to_nwb/new-code/yutanwbconverter.py [hunk 3]
```diff
@@ -13,7 +13,7 @@
 from dateutil.parser import parse as dateparse
 from pathlib import Path
 from typing import Union
-from to_nwb.neuroscope import get_clusters_single_shank, read_spike_clustering
+from neuroscope import get_clusters_single_shank, read_spike_clustering
```
Comment (bendichter) [inline buzsaki_lab_to_nwb/new-code/yutanwbconverter.py:16]:
```suggestion
from .neuroscope import get_clusters_single_shank, read_spike_clustering
```

File: buzsaki_lab_to_nwb/new-code/yutapositiondatainterface.py [hunk 4]
```diff
@@ -9,7 +9,7 @@
 from typing import Union
 # TODO: there doesn't seem to be a pypi for to_nwb...
 # we can always have them on our own end locally, but what about users?
-from to_nwb.neuroscope import add_position_data
+from neuroscope import add_position_data
```
Comment (bendichter) [inline buzsaki_lab_to_nwb/new-code/yutapositiondatainterface.py:12]:
```suggestion
from .neuroscope import add_position_data
```

File: buzsaki_lab_to_nwb/new-code/yutabehaviordatainterface.py (hunk 1 above)
Comment (CodyCBakerPhD) [reply buzsaki_lab_to_nwb/new-code/yutabehaviordatainterface.py]: Fixed

File: buzsaki_lab_to_nwb/new-code/yutanwbconverter.py (hunk 3 above)
Comment (CodyCBakerPhD) [reply buzsaki_lab_to_nwb/new-code/yutanwbconverter.py]: Fixed

File: buzsaki_lab_to_nwb/new-code/yutapositiondatainterface.py (hunk 4 above)
Comment (CodyCBakerPhD) [reply buzsaki_lab_to_nwb/new-code/yutapositiondatainterface.py]: Fixed

File: buzsaki_lab_to_nwb/new-code/yutalfpdatainterface.py (hunk 2 above)
Comment (CodyCBakerPhD) [reply buzsaki_lab_to_nwb/new-code/yutalfpdatainterface.py]: Fixed

**Pull request merged**